index.js

```diff
@@ -49,6 +49,17 @@ function filterUnusedLabels(content) {
     });
 }
 
+function filterRecvonly(content) {
+    if (!content.application) return;
+
+    // remove sources that are missing an msid (they are recvonly)
+    var sources = content.application.sources || [];
+    content.application.sources = sources.filter(function (source) {
+        // recvonly sources only have a cname
+        return source.parameters.length >= 2;
```

Comment:
Might it be worth checking explicitly for the parameters you're looking for rather than the number of parameters?